exchanges/account/account.go

```diff
@@ -396,7 +396,7 @@ func (b *ProtectedBalance) load(change *Balance) error {
 	}
 	b.m.Lock()
 	defer b.m.Unlock()
-	if !b.updatedAt.IsZero() && !b.updatedAt.Before(change.UpdatedAt) {
+	if !b.updatedAt.IsZero() && b.updatedAt.After(change.UpdatedAt) {
```

Comment:
👀 If accepted, these changes and the associated ones need to be merged into #1923

Reply:
Fixed https://github.com/gbjk/gocryptotrader/commit/72379e484